tests/unit/options/unit_tests_sender_option_parser.rb

```diff
@@ -619,6 +630,17 @@ def test_sender_option_parser_user_msg_content_list_item_value_long
     )
   end # test_sender_option_parser_user_msg_content_list_item_value_long
 
+  def test_sender_option_parser_user_content_type_msg_content_list_item_values
+    for type, param, expect in [["string", "1", "1"], ["int", "1", 1],
+                                ["float", "1", 1.0], ["bool", "true", true]]
+      assert_equal(
+          [expect],
+          Options::SenderOptionParser.new(
+              ["-L", param, "--content-type", type]).options.msg_content
+      )
+    end
+  end # test_sender_option_parser_user_content_type_msg_content_list_item_values
+
```

Comment:
Should contain tests for map, msg-property, msg-content-from-file.

And tests of exception raise when converse fails.

Reply:
done

Reply:
ok